src/public/mathfield-element.ts

```diff
@@ -337,7 +337,7 @@ export class MathfieldElement extends HTMLElement implements Mathfield {
             'virtual-keyboard-mode': 'string',
             'virtual-keyboard-theme': 'string',
             'virtual-keyboards': 'string',
-            'use-proxy-host': 'boolean',
+            'use-shared-virtual-keyboard': 'boolean',
```

Comment:
I think that target origin attribute also should be introduced there

Reply:
ah, yes, you're right